### PR TITLE
add artic page

### PR DESCRIPTION
**Pull request by DoBaHoangNam**
[Background]

create artic activity

[Side-effect]

None
[Evidence]

364399257_315948894202555_337699866665924852_n

### CODE REVIEW

File: app/src/main/java/com/example/android/ui/activity/MainActivity.kt [hunk 1]
```diff
@@ -23,80 +22,131 @@ class MainActivity : AppCompatActivity() {
...
+                LibraryFragment(),
+                R.drawable.ic_profile_selected,
+                mainBinding.tvProfile
+            )
         }
-
-
     }
 
-    private fun selectTab(tab: Int, fragment: Fragment, selectedImageId: Int, selectedTextView: TextView) {
```
Comment (NguyenKhacPhuc) [inline app/src/main/java/com/example/android/ui/activity/MainActivity.kt]: why delete this

**Comment by NguyenKhacPhuc** [comment]
resolve conflict pls

File: app/src/main/java/com/example/android/ui/activity/MainActivity.kt (hunk 1 above)
Comment (DoBaHoangNam) [reply app/src/main/java/com/example/android/ui/activity/MainActivity.kt]: ơ e có xóa đâu a nhỉ

**Review by NguyenKhacPhuc — APPROVED** [review]

**Comment by NguyenKhacPhuc** [comment]
LGTM!

**Pull request merged**